src/views/dashboard/Header.js

```diff
@@ -1,9 +1,25 @@
 import React, { Component } from 'react'
+import { withRouter } from 'react-router';
 import logoUrl from './../../assets/logo.svg'
 import userUrl from './../../assets/user.svg'
+import autobind from 'autobind-decorator';
+
 
 class Header extends Component {
+
+  @autobind
+  goToRoute(to) {
+    return () => {
+      window.focus();
```

Comment:
We have to do this weird focus stuff because we use css to control the visibility of the dropdown. Also the reason we can't use `<Link />`